src/types.ts

```diff
@@ -74,6 +74,20 @@ export interface RetailCalendar {
   numberOfWeeks: number
   months: RetailCalendarMonth[]
   weeks: RetailCalendarWeek[]
+  days: RetailCalendarDay[]
+}
+
+export interface RetailCalendarDay {
```

Comment:
@cubbuk Do you think the  fields on this interface sufficient?

Reply:
I think that would be enough for us.